source/server/server.cc

```diff
@@ -15,6 +15,7 @@
 #include "envoy/upstream/cluster_manager.h"
 
 #include "common/api/api_impl.h"
+#include "common/common/backward.h"
```

Comment:
I think we should refactor `main.cc` into site and non-site specific files, where the site-specific file is where the `main` symbol lives. It will be necessary for some site to have additional flag, logging, core dump, signal handler, memory management and other deployment specific init code that might want to run before we jump into Envoy proper (and teardown code on the way out). This can be largely opaque to the regular OSS build as long as we don't put too much code inside `main()` and can replace it with our own.